http://bugs.gentoo.org/293183

From 8cfd4a9028280e7205c7d1a70c2709706627e98d Mon Sep 17 00:00:00 2001
From: Eric Blake <ebb9@byu.net>
Date: Wed, 18 Nov 2009 20:36:01 -0700
Subject: [PATCH] tests: avoid false negative with --with-packager

Ignore extra line produced by version-etc.

* tests/test-version-etc.sh: Discard packager information.
* tests/test-argp-version-etc-1.sh: Likewise.
Reported by Mike Frysinger.

Signed-off-by: Eric Blake <ebb9@byu.net>
---
 ChangeLog                        |    5 +++++
 tests/test-argp-version-etc-1.sh |    3 ++-
 tests/test-version-etc.sh        |    3 ++-
 3 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/tests/test-version-etc.sh b/tests/test-version-etc.sh
index 528fadd..071764a 100755
--- a/gnulib-tests/test-version-etc.sh
+++ b/gnulib-tests/test-version-etc.sh
@@ -33,7 +33,8 @@ EOT
 
 ./test-version-etc${EXEEXT} --version |
  sed '1s/test-version-etc (.*) .*/test-version-etc (PROJECT) VERSION/
-      2s/Copyright (C) [0-9]\{4,4\}/COPYRIGHT/' |
+      /^Packaged by/d
+      2,3 s/Copyright (C) [0-9]\{4,4\}/COPYRIGHT/' |
  tr -d '\015' |
  diff -c $TMP - || ERR=1
 
-- 
1.6.5.2

